### PR TITLE
[docc] "Deploy your first Lambda" article

### PR DESCRIPTION
**Pull request by fabianfett**
We should have documentation on how to deploy to AWS Lambda inside the project.

- Add an article on how to deploy your first Lambda to AWS.
- This will need adjustments as soon as: #254 lands
- We will also need a new CI job that generates a docc html website and deploys it to: https://swift-server.github.io/swift-aws-lambda-runtime/

### CODE REVIEW

File: Sources/AWSLambdaRuntimeCore/Documentation.docc/Getting-started.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+struct SquareNumberHandler: LambdaHandler {
+  typealias Event = Input
+  typealias Output = Number
+
+  init(context: LambdaInitializationContext) async {
+
+  }
+
+  func handle(_ event: Input, context: LambdaContext) async throws -> Number {
+    Number(result: input.number * input.number)
```
Comment (BrianHenryIE) [inline Sources/AWSLambdaRuntimeCore/Documentation.docc/Getting-started.md:111]: This should be 
```
Number(result: event.number * event.number)
```

File: Sources/AWSLambdaRuntimeCore/Documentation.docc/Getting-started.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+Your Lambda will be executed on the Amazon Linux 2 operating system. This is why we need to compile our code for this environment. To achieve this, we use Docker.
+
+Run the following command in the root folder of your project.
+
+```bash 
+$ docker run \
+    --rm \
+    --volume "$(pwd)/:/src" \
+    --workdir "/src/" \
+    swift:5.6-amazonlinux2 \
```
Comment (BrianHenryIE) [inline Sources/AWSLambdaRuntimeCore/Documentation.docc/Getting-started.md:166]: This needs to be updated to work with `1.0.0-alpha.1`, also above in `Package.swift`.

> 'swift-aws-lambda-runtime': error: package 'swift-aws-lambda-runtime' is using Swift tools version 5.7.0 but the installed version is 5.6.3

**Comment by sebsto** [comment]
As suggested by @tomerd  and @fabianfett : I started to work on an update, on my own branch.  PR to follow.

**Comment by sebsto** [comment]
Can be closed in favour of https://github.com/swift-server/swift-aws-lambda-runtime/pull/300

**Comment by tomerd** [comment]
addressed by #316, closing

**Pull request closed without merging**